db/structure.sql

```diff
@@ -17864,3 +17865,7 @@ INSERT INTO schema_migrations (version) VALUES ('20170602144753');
 
 INSERT INTO schema_migrations (version) VALUES ('20170804101025');
 
+INSERT INTO schema_migrations (version) VALUES ('20170821140739');
```

Comment:
What is this migration version that is sneaking in the structure.sql? Could you regenerate the file to remove this? (`db:drop db:create db:migrate`)